.php_cs

```diff
@@ -9,10 +9,15 @@ $finder = PhpCsFixer\Finder::create()
 return PhpCsFixer\Config::create()
     ->setRules([
         '@PSR2' => true,
+        '@PHP70Migration' => true,
+        '@PHP70Migration:risky' => true,
+        '@PHP71Migration' => true,
+        '@PHP71Migration:risky' => true,
```

Comment:
I added all of these due to the minimum PHP version bump.

This mostly just adds the `: void` return type to methods that don't return anything.